### PR TITLE
Interactive form elements

### LINKED ISSUE #1175: Checkbox

### LINKED ISSUE #1176: Radio

### LINKED ISSUE #1177: Textinput

### LINKED ISSUE #1178: Selects

### LINKED ISSUE #1179: Textarea

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-28 at 15 56 20
screen shot 2017-07-28 at 15 56 46
screen shot 2017-07-28 at 15 56 55
screen shot 2017-07-28 at 15 57 04
screen shot 2017-07-28 at 15 57 11
screen shot 2017-07-28 at 16 01 43

Closes #1177 
Closes #1178 
Closes #1179 
Closes #1176 
Closes #1175

### CODE REVIEW

File: src/components/form-elements/Select.jsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ const optionShape = PropTypes.shape({
 });
 
 Select.propTypes = {
-  type: PropTypes.string,
```
Comment (anna-wro) [inline src/components/form-elements/Select.jsx]: this prop is not used in this component

**Comment by anna-wro** [comment]
@kdzwinel done!

File: src/components/form-elements/pages/input-interactive.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Input, {SIZE, COLOR} from '../Input';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+
+const Inputs = () => {
+  const TYPE = {
```
Comment (matikrk) [inline src/components/form-elements/pages/input-interactive.jsx]: move this to input component

Comment (anna-wro) [inline src/components/form-elements/pages/input-interactive.jsx]: moved, thanks for noticing :)

**Pull request merged**